dotenv.py

```diff
@@ -86,7 +86,12 @@ def parse_dotenv(dotenv_path):
             if not line or line.startswith('#') or '=' not in line:
                 continue
             k, v = line.split('=', 1)
-            v = v.strip("'").strip('"')
+            v = str(v).replace("'", "").replace('"', '').replace(" ", "")
```

Comment:
@Hiyorimi The logic earlier was to strip the leading quotes and don't modify the inner string of the value. so `foo="bar"` or `foo="a bar"` will be `bar` and `a bar`.  

As for the quotes inside the value or key, we shouldn't modify/remove them, as max an warning or error can be displayed.